src/resolvers.ts

```diff
@@ -48,5 +53,32 @@ export const resolvers = {
         birthDate: newUser.birthDate,
       };
     },
+
+    login: async (_parent: never, args: { data: LoginInput }) => {
+
+      const user = await dataSource.getRepository(User).findOne({
+        where: {
+          email: args.data.email,
+        },
+      });
+
+      if (!user) {
+        throw new CustomError(401, 'Falha de autenticação', 'Email não existe');
+      }
+
+      if (!(await bcrypt.compare(args.data.password, user.password))) {
+        throw new CustomError(401, 'Falha de autenticação', 'Senha incorreta');
+      }
+
+      return {
+        user: {
+          id: user.id,
+          name: user.name,
+          email: user.email,
+          birthDate: user.birthDate
+        },
```

Comment:
```suggestion
      return {
        user,
```

Aqui você pode fazer isso, porque como você declarou no schema esses 4 campos de usuário, você pode retornar todo o objeto de usuário, memso que tenham mais campos, porque o graphql filtra.